TradeItIosTicketSDK2/BrandedTableViewCell.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+class BrandedTableViewCell: UITableViewCell {
+    @IBOutlet weak var logoTitle: UIImageView!
+    @IBOutlet weak var logoWidthConstraint: NSLayoutConstraint!
+    @IBOutlet weak var altTitle: UILabel!
+
+    var adjustedLogoHeight: Double { return 14.0 }
```

Comment:
In this case is there benefits of having it to be a read-only property vs using `let` ?